tutorials/ten_minutes.ipynb

```diff
@@ -10,9 +10,9 @@
     "\n",
```

Comment:
<p>Extraplolate might not be the right word here. We apply per partition. I think extrapolate suggests we get the semantic meaning and convert?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fugue-project/tutorials/pull/150/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>Good catch, Perhaps 'shepherd' or 'marshal'?</p>